src/app/templates/map/year_control_mobile.html

```diff
@@ -0,0 +1,12 @@
+<i class="far fa-calendar-alt"></i>
+<span class="current-year"><%= current_year %></span>
+<div class="year-dropdown-control">
+    <ol>
+        <% _.each(years, function(year){ %>
+            <li>
+                <a  id="year-<%= year %>" id="year-<%= year %>" value="<%= year %>" name="year" <%=( year== current_year) ? 'checked="checked"' : ''%>><%= year %></a>
+            </li>
+        <% }) %>
+    </ol>
+</div>
+
```

Comment:
seems like you might have hard tabs set to 4, or be putting in "\t" ... see if prettier fixes this.

Reply:
Hard tabs are/were set at "2"